otri/filtering/filters/phase_filter.py

```diff
@@ -69,46 +90,53 @@ class PhaseMulFilter(PhaseFilter):
     Input:
         Single stream ordered by datetime
     Output:
-        A single stream containing n - c atoms
+        Single stream containing n - c atoms
     '''
 
-    def __init__(self, input_stream: Stream, keys_to_change: Collection[str], distance: int):
+    def __init__(self, input: str, output: str, keys_to_change: Collection[str], distance: int):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)